workspace.dsl

```diff
@@ -0,0 +1,167 @@
+workspace "Appeal service" {
+	model {
+		properties {
+			"structurizr.groupSeparator" "/"
+		}
+
+		userLocalPlanningAuthority = person "Local Planning Authority" "A local authority issuing decisions on planning applications"
+		userAppellant = person "Appellant" "A member of the public appealing against a planning decision"
+
+		group "Planning Inspectorate" {
+			userCaseOfficer = person "Case officer" "Checks LPA questionnaire and uploads statements and comments"
+			userInspector = person "Inspector" "Visits appeal site and issues appeal decision"
+
+			systemAppsFo = softwareSystem "Appeals Front-Office" "Public dashboards for appellants and Local Planning Authorities" "Node.js, Azure Web App, Azure Function App" {
+				tags "InternalCollaborationSystem"
+
+				containerFoFileStorage = container "File storage" "Stores all files uploaded in the Front-Office"
+			}
+
+			systemAppsBo = softwareSystem "Appeals Back-Office" "Internally facing service allowing for management of planning appeals in England"  {
+				tags "InternalCollaborationSystem"
+
+				group "Web" {
+
+    				containerBoWeb = container "Back-Office Web" "Back-office website" "Node.js, Azure Web App" {
+    					tags "Microsoft Azure - App Services - Web"
+    				}
+
+    				containerBoRedis = container "Redis" "Distributed session storage"
```

Comment:
As we're referencing the Azure icon theme (https://structurizr.com/help/theme?url=https://static.structurizr.com/themes/microsoft-azure-2023.01.24/icons.json) we can add tags that reference what's listed on that page

So in this case you can add a tag of `Microsoft Azure - Cache Redis` and it'll render the diagram with an Azure Redis icon here